product-scenarios/scenarios-commons/src/main/java/org/wso2/identity/scenarios/commons/util/SCIMProvisioningUtil.java

```diff
@@ -0,0 +1,116 @@
+import static org.wso2.identity.scenarios.commons.util.IdentityScenarioUtil.sendPostRequestWithJSON;
+import static org.wso2.identity.scenarios.commons.util.IdentityScenarioUtil.sendDeleteRequest;
+/**
+ *  Utility class for common functions for user management scenario.
+ */
+public class SCIMProvisioningUtil {
+
+
+    /**Headers
+     * @param username               Authenticating username.
+     * @param password               Authenticating user password.
+     */
+    private static Header[]  getHeader(String username, String password){
+        Header[] headers = {
+                new BasicHeader(HttpHeaders.CONTENT_TYPE, Constants.CONTENT_TYPE_APPLICATION_JSON),
+                new BasicHeader(HttpHeaders.AUTHORIZATION, constructBasicAuthzHeader(username, password))
+        };
+        return headers;
+    }
+
+    /**
+     * Provision a new user
+     * @param serverURL              the server url
+     * @param jsonObject             the json object of the scim payload
+     * @param username               Authenticating username.
+     * @param password               Authenticating user password.
+     * @return HttpResponse with the result.
+     * @throws IOException If error occurs during app creation.
+     */
+    public static HttpResponse provisionUserSCIM11(String serverURL, JSONObject jsonObject,
+                                                   String username, String password) throws IOException {
+
+        return provisionSCIM11Entity(serverURL,Constants.SCIM_ENDPOINT_USER, jsonObject, username, password);
+    }
+
+
+    /**
+     * Provision a new user
+     * @param serverURL              the server url
+     * @param scim11EndpointType     type ex:User,Bulk,Group
+     * @param jsonObject             the json object of the scim payload
+     * @param username               Authenticating username.
+     * @param password               Authenticating user password.
+     * @return HttpResponse with the result.
+     * @throws IOException If error occurs during app creation.
+     */
+    public static HttpResponse provisionSCIM11Entity(String serverURL, String scim11EndpointType, JSONObject jsonObject,
+                                                   String username, String password) throws IOException {
+
+        HttpClient client = HttpClients.createDefault();
+        String scimEndpoint = serverURL + Constants.SCIM_11_ENDPOINT + "/" + scim11EndpointType;
+
+        return sendPostRequestWithJSON(client, scimEndpoint, jsonObject,getHeader(username, password));
+    }
+
+    /**
+     * Delete a  user
+     * @param serverURL              the server url.
+     * @param userId                 the ID of the user to be deleted.
+     * @param username               Authenticating username.
+     * @param password               Authenticating user password.
+     * @return HttpResponse with the result.
+     * @throws IOException If error occurs during app creation.
```

Comment:
Correct the description

Reply:
thanks, fixed all lines which read "app creation"